lib/services/apache.pm

```diff
@@ -50,14 +50,13 @@ sub check_function {
 # check apache service before and after migration
 # stage is 'before' or 'after' system migration.
 sub full_apache_check {
-    my ($stage) = @_;
+    my ($stage, $type) = @_;
     $stage //= '';
     if ($stage eq 'before') {
         install_service();
-        enable_service();
-        start_service();
```

Comment:
If the start_service is no using any more, remove definition then.

Reply:
this is the new wrapper function for service start for both init.d and systemd